lib/night_in_salem_web/controllers/game_session_controller.ex

```diff
@@ -12,36 +12,37 @@ defmodule NightInSalemWeb.GameSessionController do
   end
 
   def create(conn, %{ "game_session" => %{ "player_name" => player_name } }) do
-    game_id = GameMasterSupervisor.start_game
-    GameMaster.add_player(game_id, player_name)
+    game_key = GameMasterSupervisor.start_game
+    GameMaster.add_player(game_key, player_name)
     conn
       |> put_session(:player_name, player_name)
-      |> redirect(to: game_session_path(conn, :show, game_id))
+      |> put_session(:game_key, game_key)
```

Comment:
And I should have also mentioned, +1 for the switch from game_id to game_key, I like that name better for sure.

Reply:
@Anachronomer good call, I'm only doing that in order to assign the game key in the view [here](https://github.com/ElixirCLE/night_in_salem/pull/14/files#diff-adba27a8e86cc2389163a379259dd7edR34) but I can look to see how difficult it might be to do that using the params instead.

Reply:
Also, regarding the new controller: its really in order to have a route which will let me prompt the user for a key, tbh I'm not sure if its needed but I think one of our original ideas was to give users a way to enter a key to join a game, since we are able to share a url and send that then maybe its not needed.  I could go either way, but think on it and let me know your thoughts.  If it doesn't make sense and we just want to have folks share a url that will take them to a session without entering a key by hand then that's doable, I also thought that there might be some cases where sharing a url can be difficult so it might be nice to have both options.

Reply:
I missed the use in the eex file, I see no reason to change it from that.